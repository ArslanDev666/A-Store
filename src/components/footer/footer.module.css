.footer {
  padding-bottom: var(--gap-6xl);

  color: var(--color-text-black-alpha-20);
}