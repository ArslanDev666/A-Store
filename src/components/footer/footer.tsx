import React from 'react';

import { Typography } from '@alfalab/core-components/typography';

import { Container } from 'components/ui/container';

import styles from './footer.module.css';

const CURRENT_YEAR = new Date().getFullYear();

const Footer = () => {
  return (
    <footer className={styles.footer}>
      <Container>
        <Typography.Text weight='medium'>
          © ООО «Альфа Фьюче Пипл», {CURRENT_YEAR}
        </Typography.Text>
      </Container>
    </footer>
  );
};

export { Footer };
