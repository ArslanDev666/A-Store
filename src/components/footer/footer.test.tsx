import renderer from 'react-test-renderer';

import { Footer } from './footer';

describe('Footer component', () => {
  describe('Render tests', () => {
    it('should render correctly', () => {
      const tree = renderer.create(<Footer />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
