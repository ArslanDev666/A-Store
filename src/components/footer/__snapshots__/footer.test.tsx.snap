// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component Render tests should render correctly 1`] = `
<footer
  className="footer"
  data-test-id="footer"
>
  <div
    className="container"
    data-test-id="container"
  >
    <span
      className="typography__primary-medium_xcdcz typography__medium_xcdcz"
    >
      © ООО «Альфа Фьюче Пипл», 
      2023
    </span>
  </div>
</footer>
`;
